README.md

```diff
@@ -66,34 +66,40 @@ Install the ``python3-routes`` package:
 
 These instructions are meant for developers who are compiling the code for
 development and testing. To build binaries that are suitable for installation
-we recommend that you build .deb or .rpm packages, or refer to ``ceph.spec.in``
-or ``debian/rules`` to see which configuration options are specified for
-production builds.
+we recommend that you build `.deb` or `.rpm` packages, or refer to
+``ceph.spec.in`` or ``debian/rules`` to see which configuration options are
+specified for production builds.
 
-Build instructions:
+To build ceph, make sure that you are in the top-level `/ceph` directory, and
+run the following commands:
 
 	./do_cmake.sh
 	cd build
 	ninja
 
-``do_cmake.sh`` defaults to creating a debug build of Ceph that can be up to 5x
-slower with some workloads. Pass ``-DCMAKE_BUILD_TYPE=RelWithDebInfo`` to
+``do_cmake.sh`` by default creates a "debug build" of Ceph, which can be up to
+5x slower with some workloads. Pass ``-DCMAKE_BUILD_TYPE=RelWithDebInfo`` to
 ``do_cmake.sh`` to create a non-debug release.
 
 The number of jobs used by `ninja` is derived from the number of CPU cores of
 the building host if unspecified. Use the `-j` option to limit the job number
-if the build jobs are running out of memory. On average, each job takes around
+if the build jobs are running out of memory. If you attempt to run `ninja` and
+receive a message that reads `g++: fatal error: Killed signal terminated
+program cc1plus`, then you have run out of memory. Using the `-j` option with
+an argument appropriate to the hardware on which the ninja command is run is
+expected to result in a successful build. For example, to limit the job number
```

Comment:
the `ninja` command is run should result in

Reply:
In this case, I made the decision to cut out the modal "should", because I think it can be read as optative. What I want is for this sentence to be absolutely declarative.